From 69619288290ceac4e5795b382c29f171ac2cf0c3 Mon Sep 17 00:00:00 2001
From: Gao Pan <pandy.gao@nxp.com>
Date: Wed, 9 Aug 2017 16:47:05 +0800
Subject: [PATCH 2368/5242] MLK-16189 arm64: dts: enable i2c bus expander for
 imx8qxp-mek

commit  7db9cf37f7e3f60e42a863e6acb8b7f8dcfb7034 from
https://source.codeaurora.org/external/imx/linux-imx.git

enable i2c bus expander for imx8qxp-mek

Signed-off-by: Gao Pan <pandy.gao@nxp.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 arch/arm64/boot/dts/freescale/fsl-imx8qxp-mek.dts |    9 ++++++++-
 1 file changed, 8 insertions(+), 1 deletion(-)

diff --git a/arch/arm64/boot/dts/freescale/fsl-imx8qxp-mek.dts b/arch/arm64/boot/dts/freescale/fsl-imx8qxp-mek.dts
index bb79c9428..9603758 100644
--- a/arch/arm64/boot/dts/freescale/fsl-imx8qxp-mek.dts
+++ b/arch/arm64/boot/dts/freescale/fsl-imx8qxp-mek.dts
@@ -128,6 +128,12 @@
 			>;
 		};
 
+		pinctrl_ioexp_rst: ioexp_rst_grp {
+			fsl,pins = <
+				SC_P_SPI2_SDO_LSIO_GPIO1_IO01	0x06000021
+			>;
+		};
+
 		pinctrl_isl29023: isl29023grp {
 			fsl,pins = <
 				SC_P_SPI2_SDI_LSIO_GPIO1_IO02	0x00000021
@@ -318,7 +324,8 @@
 	#size-cells = <0>;
 	clock-frequency = <100000>;
 	pinctrl-names = "default";
-	pinctrl-0 = <&pinctrl_lpi2c1>;
+	pinctrl-0 = <&pinctrl_lpi2c1 &pinctrl_ioexp_rst>;
+	pinctrl-assert-gpios = <&gpio1 1 GPIO_ACTIVE_HIGH>;
 	status = "okay";
 
 	pca9646@71 {
-- 
1.7.9.5

